paramiko/buffered_pipe.py

```diff
@@ -101,7 +101,7 @@ def feed(self, data):
             if self._event is not None:
                 self._event.set()
             self._buffer_frombytes(b(data))
-            self._cv.notifyAll()
+            notify_all(self._cv)
```

Comment:
We could use simply `self._cv.notify_all()` with Python 2.7 and 3.

* https://docs.python.org/2/library/threading.html#threading.Condition.notify_all
* https://docs.python.org/3/library/threading.html#threading.Condition.notify_all